code/game/objects/structures/mirror.dm

```diff
@@ -38,7 +53,19 @@
 	..()
 
 
-/obj/structure/mirror/attackby(obj/item/I as obj, mob/living/user as mob, params)
+/obj/structure/mirror/attackby(obj/item/I, mob/living/user, params)
+	if(isscrewdriver(I))
+		visible_message("<span class='notice'>[user] begins to unfasten the [src].</span>", "<span class='notice'>You begin to unfasten the [name].</span>")

```

Comment:
Drop `the` before `[src]`

Reply:
Also, change the second message from `[name]` to `[src]` and drop `the` there as well

Reply:
None of these visual messages actually work, by the way- You're calling `visible_message` on the mirror, you would have to call `user.visible_message` in order to use the second argument